### PR TITLE
Add input file

### PR DESCRIPTION
**Pull request by HichamELBSI**
Signed-off-by: HichamELBSI <elabbassih@gmail.com>

### CODE REVIEW

File: packages/strapi-plugin-upload/admin/src/components/InputMedia/EmptyText.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import IntlText from '../IntlText';
+
+const EmptyText = styled(IntlText)`
+  color: ${({ theme }) => theme.main.colors.white};
+  position: absolute;
+  width: 17rem;
+  text-align: center;
```
Comment (virginieky) [inline packages/strapi-plugin-upload/admin/src/components/InputMedia/EmptyText.js]: Can you please order the attributes?

```
  position: absolute;
  width: 17rem;
  text-align: center;
  color: ${({ theme }) => theme.main.colors.white};
```

File: packages/strapi-plugin-upload/admin/src/components/InputMedia/EmptyText.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import IntlText from '../IntlText';
+
+const EmptyText = styled(IntlText)`
+  color: ${({ theme }) => theme.main.colors.white};
+  position: absolute;
```
Comment (soupette) [inline packages/strapi-plugin-upload/admin/src/components/InputMedia/EmptyText.js]: Can you order your css?

File: packages/strapi-plugin-upload/admin/src/components/InputMedia/IconUpload.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+const IconUpload = () => {
+  return (
+    <svg
+      width="105"
+      height="84"
+      xmlns="http://www.w3.org/2000/svg"
+      xmlnsXlink="http://www.w3.org/1999/xlink"
+      style={{ margin: 'auto' }}
```
Comment (virginieky) [inline packages/strapi-plugin-upload/admin/src/components/InputMedia/IconUpload.js]: Do we really want to set this style attribute here and not set it with a styled-component compo?

File: packages/strapi-plugin-upload/admin/src/components/InputMedia/index.js [hunk 4]
```diff
@@ -1,30 +1,74 @@
...
   return (
     <Wrapper>
-      <Name htmlFor={name}>{label}</Name>
-      <CardPreviewWrapper onClick={handleClickToggleModal}>
-        <CardPreview url="https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&h=350" />
+      <Name htmlFor={name}>{`${label}${displaySlidePagination}`}</Name>
+
+      <CardPreviewWrapper>
+        <CardControlWrapper>
+          <CardControl color="#9EA7B8" type="plus" onClick={() => handleClickToggleModal(value)} />
```
Comment (virginieky) [inline packages/strapi-plugin-upload/admin/src/components/InputMedia/index.js]: I think the handleClickToggleModal() should also be on the EmptyInputMedia to open the modal on the entire empty input

File: packages/strapi-plugin-upload/admin/src/components/InputMedia/IconUpload.js (hunk 3 above)
Comment (HichamELBSI) [reply packages/strapi-plugin-upload/admin/src/components/InputMedia/IconUpload.js]: This component was in the strapi-admin package. Anyway, this inline style is useless here. Nice catch.

File: packages/strapi-plugin-upload/admin/src/components/InputMedia/index.js (hunk 4 above)
Comment (HichamELBSI) [reply packages/strapi-plugin-upload/admin/src/components/InputMedia/index.js]: Indeed

**Pull request merged**